netlify.toml

```diff
@@ -10,7 +10,7 @@ base = ""
 command = """
 pip install --upgrade pip \
 && pip install --upgrade --upgrade-strategy eager  -e ".[dev]" \
-&& pip install git+https://oauth:${MKDOCS_MATERIAL_INSIDERS_REPO_RO}@github.com/PrefectHQ/mkdocs-material-insiders.git@master \
+&& pip install git+https://oauth:${MKDOCS_MATERIAL_INSIDERS_REPO_RO}@github.com/PrefectHQ/mkdocs-material-insiders.git@a1c87bd61619694c6b2375108afc5d072fbe4b07 \
```

Comment:
this pin is necessary due to a python 3.8 incompatible type annotation in mkdocs insiders; it appears to have been fixed upstream but we haven't incorporated the fix yet - I'll remove the pin once the relevant update has been merged into our fork